packages/core/src/payment/strategies/squarev2/squarev2-payment-processor.ts

```diff
@@ -0,0 +1,159 @@
+import { BillingContact, Card, CardInputEvent, CardInputEventTypes, ChargeVerifyBuyerDetails, Payments, SqEvent } from './types';
+import { fromEvent, merge, Subscription } from 'rxjs';
+import SquareV2ScriptLoader from './squarev2-script-loader';
+import { CheckoutStore } from '../../../checkout';
+import { NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { guard } from '../../../../src/common/utility';
+import SquareV2PaymentInitializeOptions from './squarev2-payment-initialize-options';
+import { distinctUntilChanged, map } from 'rxjs/operators';
+import { BillingAddress } from '../../../../src/billing';
```

Comment:
I believe `eslint` should prompt you to rearrange the import sorting order.
Could you please follow it?

Reply:
I was expecting that too but, I'm not getting any advice about that rules. I also manually ran the `npm run lint` command and nothing was prompted about import sorting 🤷🏻‍♂️ I did the changes anyway based on `checkout-js` rules as it is working as expected for me.

Reply:
Thanks. I was thinking you should be able to see something like this.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/88361607/196830832-93a59d23-ac54-432a-88af-dce0ab46d3ad.png">

Reply:
Yeah, my linter works fine on `checkout-js` but not with this project 🤔